modules/RTC/ortc/RTCPeerConnection.js

```diff
@@ -0,0 +1,532 @@
+/* global __filename, RTCIceGatherer, RTCIceTransport */
+
+import { getLogger } from 'jitsi-meet-logger';
+import yaeti from 'yaeti';
+import { InvalidStateError } from './errors';
```

Comment:
We have some guidelines in the jitsi-meet repo. Imports are separated in groups.